src/sentry/grouping/enhancer/__init__.py

```diff
@@ -19,7 +18,7 @@
 from sentry_ophio.enhancers import Cache as RustCache
 from sentry_ophio.enhancers import Enhancements as RustEnhancements
 
-from sentry import options, projectoptions
+from sentry import projectoptions
 from sentry.features.rollout import in_random_rollout
```

Comment:
If we remove `prefer_rust_enhancers` we can remove this import.

Reply:
Indeed, I completely missed this.